Release v1.4.2

argenos opened the pull request:
## Changelog
### Enhancements
* Add Kaldi as the default speech recognition engine
* Adds the following maps:
    * German Open Arena
    * Rheinbach
* Add "Find my mates" robocup scenario
    * Add the find people action
    * Add interview person behaviour
    * Add describe person behaviour
* General improvements on "Where is this" scenario
    * Add service requests to [topological_map](https://github.com/b-it-bots/topological_map)
* `move_arm_action`
    * Replaced the DMP service call with an API call
    * All DMP parameters are now configurable 
### Fixes
* General clean up of new robocup scenarios
* Move NLU and speech models to [`mas_models`](https://github.com/b-it-bots/mas_models)
* Update dependency to [`mas_navigation_tools`](https://github.com/b-it-bots/mas_navigation_tools)
* Adds dependency to [`ros_dmp](https://github.com/b-it-bots/ros_dmp) for `mas_move_arm_action`
* All navigation goals are now lower case to follow convention

robertocaiwu committed:
[mdr_listen_action] Add kaldi functionality

robertocaiwu committed:
[mdr_listen_action] Set kaldi as default SR engine

robertocaiwu committed:
[mdr_listen_action] Remove duplicate code

minhnh committed:
add classify image as an optional param

argenos committed:
Add GO 2019 map

minhnh committed:
Merge branch 'go-2019' of https://github.com/b-it-bots/mas_domestic_robotics into go-2019

alex-mitrevski committed:
[mdr_environments] Added a GO 2019 testing map (close to tables)

argenos committed:
[mdr_environments] Add navigation goals to test map

alex-mitrevski committed:
[mdr_environments/go-small-2019] Cleaned the map a bit

alex-mitrevski committed:
Merge branch 'go-2019' of github.com:b-it-bots/mas_domestic_robotics into go-2019

Commit:
Add the "find_my_mates" scenario and "interview action".

This commit adds a first version of the "find my mates scenario of the RoboCup @home league 2019. Additionally an action for interviewing people is added. Also here it is not the final version.

PatrickNa committed:
Merge pull request #129 from PatrickNa/mdr_robocup_tasks/find_my_mates

Add the "find_my_mates" scenario and "interview action".

alex-mitrevski committed:
Merge pull request #128 from minhnh/go-2019

add classify image as an optional param

robertocaiwu committed:
[mdr_listen_action] Move ros param to listen client

Commit:
Fix merge conflic comments.

Apperently a merge conflict was in the code and that is fixed with this commit.

PatrickNa committed:
Merge pull request #131 from PatrickNa/mdr_robocup_tasks/find_my_mates

Fix merge conflic comments.

robertocaiwu committed:
[mdr_listen_action] Change default model_directory in launch file

PatrickNa committed:
Merge pull request #132 from robertocaiwu/feature/Kaldi_listen_action

Feature/kaldi listen action

argenos committed:
[mdr_environments] Add find my mates pose

Commit:
Update the "Find my mates" state machine.

This commit provides a comphrehensive version of the "Find my mate" scenario. A few more information need to be added though. For example location values.

Commit:
Add HRI behaviours that are needed in "find my mate" scenario among others.

At the moment the following behaviours are needed that are defined in the "Find my mate" scenario.

PatrickNa committed:
Merge pull request #133 from PatrickNa/mdr_robocup_tasks/find_my_mates

Update the "Find my mates" state machine.

henrikschnor committed:
Added mdr_find_people action

alex-mitrevski committed:
[rosplan.launch] Added ontology params

robertocaiwu committed:
[mdr_listen_action] Fix namespace issue in launch file

robertocaiwu committed:
[mdr_listen_action] Send ros params to smach state userdata

robertocaiwu committed:
Merge branch 'go-2019' of github.com:b-it-bots/mas_domestic_robotics into feature/Kaldi_listen_action

alex-mitrevski committed:
[default_domain.pddl] Added a person type and known/unknown predicates

alex-mitrevski committed:
[find_people_client] The KB is now updated after finding people

argenos committed:
Merge pull request #134 from robertocaiwu/feature/Kaldi_listen_action

Feature/kaldi listen action

alex-mitrevski committed:
[find_my_mates] Added states for moving to a person and checking if there are unknown people left

alex-mitrevski committed:
Merge branch 'go-2019' of github.com:b-it-bots/mas_domestic_robotics into go-2019

alex-mitrevski committed:
[find_my_mates/move_to_person] The robot is now going to the pose specified as 'safe_pose'

alex-mitrevski committed:
[find_my_mates/config] GO_TO_OPERATOR now goes back to the FIND_PEOPLE pose

Commit:
Add "Interview Person" behaviours.

This commit defines the "Interview Person" behaviour. The only thing that is missing is saving the information to the knowledge base and the specification of the model.

PatrickNa committed:
Merge pull request #135 from PatrickNa/mdr_robocup_tasks/find_my_mates

Add "Interview Person" behaviours.

alex-mitrevski committed:
[perception_msgs] Added a PersonInfo msg

henrikschnor committed:
Added arena bounds check and a safe pose to mdr_find_people

robertocaiwu committed:
[mdr_find_my_mates] Add report information behavior

Commit:
Add a RASA NLU model for the "Find My Mate" Scenario.

This commit adds a RASA NLU model, that allows to introduce yourself or to ask the robot to find certain people.

PatrickNa committed:
Merge pull request #137 from PatrickNa/mdr_robocup_tasks/find_my_mates

Add a RASA NLU model for the "Find My Mate" Scenario.

PatrickNa committed:
Merge pull request #136 from robertocaiwu/feature/mdr_find_my_mates/report

[mdr_find_my_mates] Add report information behavior

minhnh committed:
add describe person behavior

henrikschnor committed:
Included an image of detected people in the result of mdr_find_people

alex-mitrevski committed:
[find_my_mates] Moved states to the scenario package; added interaction with the KB

robertocaiwu committed:
[mdr_find_my_mates] Add retrieve_information behavior

alex-mitrevski committed:
[perception_behaviours] Added a find_people behaviour

alex-mitrevski committed:
[find_my_mates] Various error fixes

alex-mitrevski committed:
[find_my_mates/interview] Fixed an incorrect elif condition

alex-mitrevski committed:
[hri_behaviours/retrieve] Fixed various errors

mas-hsr committed:
Various fixes to the find_my_mates scenario

argenos committed:
[mdr_robocup_tasks] Add where is this pkg

argenos committed:
[mdr_robocup_tasks] Add where is this pkg

argenos committed:
[mdr_robocup_tasks] Remove scanning state

argenos committed:
[mdr_perception_msgs] Add initial ObjectInfo msg

Commit:
Merge branch 'go-2019' of github.com:b-it-bots/mas_domestic_robotics into go-2019

Commit:
Add the state machine for the "where is this" scenario of the RoboCup 2019.

This commit specifies the state machine of the "Where is this" scenario. It defines the states and transitions. However it lacks of certain specifications yet, like for example the actions that are executed within the states.

Commit:
Add the scenario states/ actions to the state machine of "where is this".

This commit adds mainly dummy scenario states for the "where is this" scenario. Also those reference are defined in the state machine now. In the upcoming commits the logic within the states is going to be defined further.

Commit:
Add remaining states (without proper functioning yet).

This commit adds the states of the scenario. At the moment all of them return succeeded so that the overall state machine can be tested.

Commit:
Update the where is this launch file.

This commit updates the launch file of the scneario, but keeps it basic for now.

robertocaiwu committed:
[mdr_simple_listen_action] Make new listen action based on FTSM

robertocaiwu committed:
Merge branch 'go-2019' of github.com:b-it-bots/mas_domestic_robotics into feature/mdr_simple_listen_action

robertocaiwu committed:
[mdr_listen_action] Refactor mdr_simple_listen_action to mdr_listen_action

mas-hsr committed:
fix errors in SM mdr_where_is_this

henrikschnor committed:
Added receive_information state

ErickKramer committed:
Added trained model of rasa nlu

ErickKramer committed:
Merge branch 'go-2019' of https://github.com/b-it-bots/mas_domestic_robotics into go-2019

mas-hsr committed:
fix issues with speech recog in receive info state

minhnh committed:
Merge pull request #139 from robertocaiwu/feature/mdr_simple_listen_action

Feature/mdr simple listen action

alex-mitrevski committed:
[where_is_this] Removed unnecessary states from the state machine

alex-mitrevski committed:
Merge branch 'go-2019' of github.com:b-it-bots/mas_domestic_robotics into go-2019

ErickKramer committed:
Added more sentences for the model

ErickKramer committed:
Merge branch 'go-2019' of https://github.com/b-it-bots/mas_domestic_robotics into go-2019

ErickKramer committed:
Added entities text file for question matchin

ErickKramer committed:
Updated rasa nlu model

ErickKramer committed:
Added locations and objects text files

Commit:
Add the speech matching component to receive information state

This commit extends the mdr_where_is_this scenario by comparing the detected entities from the operator input with the entities (objects and locations) that are predefined.

mas-hsr committed:
Fix bugs regarding the speech matching changes in receive information.

This commit provides smaller changes to the receive information state. The changes are fixes regarding the latest changes to the state. The commit also includes the modified lists of objects and locations.

mas-hsr committed:
Added logging node for speech recognition and nlu

argenos committed:
[mdr_robocup_tasks] Add topological map launch file to where_is_this

henrikschnor committed:
[mdr_where_is_this] Added detect_person state

alex-mitrevski committed:
[where_is_this] Initial draft of the describe_location state

alex-mitrevski committed:
[where_is_this/describe_location] Skipping detailed description if no objects next to

mas-hsr committed:
[mdr_where_is_this] Small fixes in detect_person state

alex-mitrevski committed:
[where_is_this] Fixed incorrect import of topological_map

alex-mitrevski committed:
Merge branch 'go-2019' of github.com:b-it-bots/mas_domestic_robotics into go-2019

alex-mitrevski committed:
[where_is_this/describe_location] Added timeout for pose

alex-mitrevski committed:
[where_is_this/describe_location] Fixed an incorrect parameter name

alex-mitrevski committed:
[where_is_this/describe_location] Fixed userdata inconsistencies

alex-mitrevski committed:
[where_is_this/describe_location] Location entities are now processsed as well

alex-mitrevski committed:
[where_is_this/describe_location] Incorrect object was passed to the top path planner

alex-mitrevski committed:
[where_is_this/describe_location] Changed the pose topic name

ErickKramer committed:
Added locations and objects relevant for the competition, removed unused ones

ErickKramer committed:
Added locations and objects relevant for the competition, removed unused ones

ErickKramer committed:
Fixed merge issue

ErickKramer committed:
Added sentences to the nlu model

argenos committed:
Merge branch 'devel' into go-2019

argenos committed:
Merge branch 'go-2019' of github.com:b-it-bots/mas_domestic_robotics into go-2019

argenos committed:
Remove go-small-2019 map

argenos committed:
Add BRSU rheinbach map

argenos committed:
Add launcher to visualize map in rviz

argenos committed:
[rosinstall] Add the topological_map repo

argenos committed:
Merge branch 'devel' into go-2019

argenos committed:
Update dependencies to mas_perception_msgs

argenos committed:
[where_is_this] Clean up comments

argenos committed:
[where_is_this] Add missing dependencies

argenos committed:
Merge branch 'devel' into go-2019

argenos committed:
Merge branch 'devel' into go-2019

minhnh committed:
fix parameter type, remove TODO

minhnh committed:
[mdr_mbot_interface] remove package.xml comments

minhnh committed:
[mdr_mbot_interface] change maintainer name

mas-hsr committed:
[speech_matching/objects] Added bookshelf to the list

mas-hsr committed:
Merge branch 'go-2019' of https://github.com/b-it-bots/mas_domestic_robotics into go-2019

mas-hsr committed:
[speech_matching/locations] Added hallway, entrance, and exit to the list

mas-hsr committed:
[where_is_this/describe_location] Bug fixed and improvements in the description

mas-hsr committed:
[where_is_this_sm] Changed the transitions so that the SM doesn't terminate

mas-hsr committed:
[perceive_plane_action/client_test] Corrected the name of the action server

mas-hsr committed:
[rosplan.launch] Specified a default path for problem and plan files

mas-hsr committed:
[mbot_logger] Removed a log file

ErickKramer committed:
Removed unnecessary comments

ErickKramer committed:
Added comments about what it is being included inside the launch file.

mas-hsr committed:
[pickup_action] Added params for low pregrasp

mas-hsr committed:
[place_action] Added params for low surface placing

Vanargh committed:
Removed unused import String

divindevaiah committed:
Removed used imported sys, yaml, actionlib, ListenFeedback

ZuhaKarim committed:
Deleted spaces and unused packages

agrija9 committed:
Codacy fixes

rkutty1 committed:
codacy fixes in interview_person.py

IshanChaks91 committed:
removed line 5 (from std_msg.msg import String) removed line 7 (from mdr_perception_msgs.msg import PersonInfo) removed line 8(from rasa_nlu.training_data import load_data) removed line 9 (from rasa_nlu import config) removed line 10 (from rasa_nlu.components import ComponentBuilder) removed Trainer imported from rasa_nlu.model (from rasa_nlu.model import Interpreter, Metadata, Trainer)

ZuhaKarim committed:
indented

divindevaiah committed:
Removed unused String library

Vanargh committed:
Removed unused TurnBaseToFeedback package

agrija9 committed:
codacy fix action_states

divindevaiah committed:
Removed unused copy, rospkg, String, ActionSlot Library

argenos committed:
Merge pull request #157 from b-it-bots/refactor/clean_mdr_mbot_logger

Refactor/clean mdr mbot logger

Vanargh committed:
Removed Unused RecognizeEmotionFeedback package

divindevaiah committed:
Removed copy, rospkg,ActionSlot and Whitespace.

agrija9 committed:
Unused PlaceGoal imported from mdr_place_action.msg

ZuhaKarim committed:
unused CVBridgerror, GenderRecognitonFeedback,number_of_faces, rgb_cvs

divindevaiah committed:
Removed unused sys Library.

IshanChaks91 committed:
removed line 5 (from std_msg.msg import String) removed line 7 (from mdr_perception_msgs.msg import PersonInfo) removed line 8(from rasa_nlu.training_data import load_data) removed line 9 (from rasa_nlu import config) removed line 10 (from rasa_nlu.components import ComponentBuilder) removed Metadata, Trainer imported from rasa_nlu.model (from rasa_nlu.model import Interpreter, Metadata, Trainer)

Vanargh committed:
Removed Unused packages rospy  tf PointCloud2 transform_cloud_with_listener

rkutty1 committed:
codacy fix

argenos committed:
Merge pull request #161 from rkutty1/go-2019

codacy fixes in interview_person

divindevaiah committed:
Removed unused EnterDoorFeedback Library.

agrija9 committed:
Fixed Unused import smach_ros

divindevaiah committed:
Removed unused MoveBaseFeedback Library.

Vanargh committed:
Removed unused config in retrieve_information and load_data in interview_person

IshanChaks91 committed:
removed line 4 (import geometry_msgs.msg) removed whitespace betwee line 89 and 90(trailing whitespace)

agrija9 committed:
Fixed unused import actionlib

divindevaiah committed:
Removed unused PickupFeedback.

divindevaiah committed:
Removed unused MoveForwardFeedback.

agrija9 committed:
Fixed unused import sys

IshanChaks91 committed:
removed MoveArmFeedback from line 9 (from mdr_move_arm_action.msg import MoveArmGoal, MoveArmFeedback, MoveArmResult)

Vanargh committed:
Merge branch 'go-2019' of https://github.com/b-it-bots/mas_domestic_robotics into go-2019

divindevaiah committed:
Removed unused PerceivePlaneFeedback.

mas-hsr committed:
[environments/brsu-c069] Updated navigation goals

mas-hsr committed:
Merge branch 'go-2019' of https://github.com/b-it-bots/mas_domestic_robotics into go-2019

mas-hsr committed:
[demo_patrol_sm] Updated the pose names

divindevaiah committed:
Removed unused randint.

ZuhaKarim committed:
codacy review

argenos committed:
Merge pull request #165 from rkutty1/go-2019

codacy fix

argenos committed:
Merge pull request #160 from Vanargh/go-2019

Codacy Fixes

mas-hsr committed:
[store_groceries] Added a minimal SM specification for running lab tests

mas-hsr committed:
Merge branch 'go-2019' of https://github.com/b-it-bots/mas_domestic_robotics into go-2019

mas-hsr committed:
[where_is_this] Added a minimal SM for lab testing

mas-hsr committed:
[robot_inspection_sm] Removed unnecessary states

mas-hsr committed:
[robot_inspection] Added an SM for lab testing

argenos committed:
Merge pull request #164 from divindevaiah/go-2019

Codacy cleanup in several packages

argenos committed:
Merge pull request #166 from ZuhaKarim/go-2019

Codacy cleanup in various packages

argenos committed:
Update mdr_hri/mdr_mbot_logger/ros/src/mdr_mbot_logger/mbot_logger.py

argenos committed:
Update mdr_hri/mdr_mbot_logger/ros/src/mdr_mbot_logger/mbot_logger.py

argenos committed:
Merge pull request #162 from agrija9/go-2019

Codacy fixes

alex-mitrevski committed:
Merge branch 'go-2019' into go-2019

alex-mitrevski committed:
Merge pull request #163 from IshanChaks91/go-2019

Modified retrieve_information.py in go-2019 mdr_planning/mdr_behaviours/mdr_hri_behaviours/ros/srs/mdr_hri_behaiours

argenos committed:
Merge pull request #145 from b-it-bots/go-2019

Adding new robocup scenarios

argenos committed:
Merge branch 'kinetic' into devel

alex-mitrevski committed:
[environments/rheinbach] Lower case nav goals

alex-mitrevski committed:
[environments/go-2019] Lower case nav goals

argenos committed:
Merge pull request #169 from b-it-bots/refactor/lower-case-nav-goals

Lower case nav goals

alex-mitrevski commented on mdr_hri/mdr_mbot_logger/package.xml:
```diff
@@ -0,0 +1,20 @@
+<package format="2">
+  <name>mdr_mbot_logger</name>
+  <version>0.0.0</version>
+  <description>The mdr_mbot_logger package</description>
```
@ErickKramer What is the actual purpose of this package? I assume it's used for data collection and model evaluation? Could you please update the description accordingly?

ErickKramer replied:
The purpose of the package is to log the sentences recognized by the speech_recognizer and the interpretations generated by mbot. 

Sure, I will update the description.

minhnh replied:
resolved in b967c47

alex-mitrevski committed:
[find_people_action] Added install targets to the CMakeLists

alex-mitrevski commented on mdr_planning/mdr_actions/mdr_perception_actions/mdr_find_people/ros/src/mdr_find_people/action_states.py:
```diff
@@ -0,0 +1,111 @@
+POINTCLOUD_TOPIC = '/hsrb/head_rgbd_sensor/depth_registered/rectified_points'
+
+
+class FindPeopleState(smach.State):
```
This should be reimplemented using FTSM.

argenos replied:
Can you add this to #172 ?

alex-mitrevski committed:
[hri_behaviours] CMakeLists and package.xml cleanup

alex-mitrevski committed:
[find_my_mates_sm] Lower case pose names

alex-mitrevski commented on mdr_speech/mdr_speech_matching/common/config/entities.txt:
```diff
@@ -0,0 +1,259 @@
+senbei
```
The contents of `entities.txt`, `locations.txt`, and `objects.txt` should all be generated from the ontology on the fly (so, first of all, the contents should be encoded in the ontology and second of all, they should be retrieved as necessary).

PatrickNa replied:
It depends. Sometimes you want to compare the recognized speech to a specific list of objects... Generating it from the ontology fills the list with words that are not of interest.

alex-mitrevski replied:
That's only the case if one takes *everything* from the ontology, which doesn't have to be the case. E.g. if only kitchen items are relevant for a specific context, those are the ones to retrieve and compare against.

PatrickNa replied:
I would say both options are valid and should be taken care of. Let's open an issue?!

alex-mitrevski replied:
Yeah, we should think about an issue. This is definitely not something that should be done for this PR, I just wanted to open the discussion so that we have that in mind. But I guess we should first discuss this in more detail so that we have a better idea on what someone working on the issue should do.

argenos replied:
Is this related? https://github.com/b-it-bots/mas_knowledge_base/issues/2

alex-mitrevski replied:
No, not really. This is in some sense an inverse of that (although technically that's also not fully correct because these entities will be taken from the TBox rather than the ABox since they represent the classes of objects, not the specific instances).

argenos committed:
Remove repo-specific templates

argenos committed:
mdr_environments: Update dependency to mas_navigation_tools

robertocaiwu committed:
[mdr_listen_action] Remove hsr specific parameter from launch file

robertocaiwu committed:
[mdr_speech_recognition] Add Kaldi models folder to gitignore

alex-mitrevski commented:
@ErickKramer Could you please (re)move the file `mdr_planning/mdr_scenarios/mdr_robocup_tasks/mdr_where_is_this/common/model/current/nlu/training_data.json`, which has 150,000> lines?

ErickKramer commented:
@alex-mitrevski that file is automatically generated by the training script from rasa. I tested on my computer and the interpreter works without it, so probably it just uses it during training. Should I remove it then?.

argenos commented:
@ErickKramer yes, please move everything related to the models to [mas_models](https://github.com/b-it-bots/mas_models). Let's continue the discussion of the clean up in #174.

ErickKramer committed:
Added a more describtive description to the package.xml file of mdr_mbot_logger

ErickKramer committed:
Added a more describtive description to the package.xml file of mdr_mbot_logger

minhnh commented on .gitignore:
```diff
@@ -16,3 +16,6 @@ mongod.lock
 message_store.*
+
+# Kaldi models
+/mdr_speech/mdr_speech_recognition/config/models/*
```
Assuming the models to be moved to [`b-it-bots/mas_models`](https://github.com/b-it-bots/mas_models), this may not be necessary. We may also be interested in keeping track of the model changes, but maybe use LFS instead of regular files.

argenos replied:
Indeed the speech models need to be moved. See #179 

cc @robertocaiwu

sthoduka committed:
fix pose_visualiser spelling in launch file

argenos committed:
Merge pull request #181 from sthoduka/fix/spelling

fix pose_visualiser spelling in launch file

alex-mitrevski committed:
[docs] Updated the architecture diagram

argenos committed:
Travis: Only run sonarclouds checks if not a fork

PatrickNa commented on mdr_planning/mdr_actions/mdr_speech_actions/mdr_listen_action/ros/scripts/listen:
```diff
@@ -3,67 +3,49 @@
 """
 Created on 2018.10.09
 
-@author: patrick
```
LOL

alex-mitrevski committed:
[mdr_actions] Added a README

alex-mitrevski committed:
[store_groceries] Fixed various consistency issues with the SM

alex-mitrevski committed:
[find_my_mates] Fixed verification issues with the SM

alex-mitrevski committed:
[spr_sm] Removed an unused state from the state list

alex-mitrevski committed:
[where_is_this_sm_lab] Removed an undefined state from the state list

ErickKramer committed:
Merge branch 'devel' of https://github.com/b-it-bots/mas_domestic_robotics into devel

alex-mitrevski committed:
[demo_simple_pick_and_place] Changed all nav goals to lowercase letters

alex-mitrevski committed:
[demo_throw_table_objects] Changed all nav goals to lowercase letters

alex-mitrevski committed:
[serve_drinks_sm] Changed all nav goals to lowercase letters

alex-mitrevski committed:
[take_out_garbage_sm] Changed all nav goals to lowercase letters

minhnh commented on mdr_planning/mdr_scenarios/mdr_robocup_tasks/mdr_where_is_this/CMakeLists.txt:
```diff
@@ -0,0 +1,35 @@
+project(mdr_where_is_this)
+
+find_package(catkin REQUIRED COMPONENTS
+  rospy
```
I believe there's missing dependency on `mas_knowledge_base`

alex-mitrevski replied:
Added with commit https://github.com/b-it-bots/mas_domestic_robotics/commit/5ee38a9dfa0771e085e4a3f61b08e9380ac15379

alex-mitrevski committed:
[where_is_this] Updated the author list and added mas_knowledge_base to the dependency list

alex-mitrevski committed:
[find_my_mates/move_to_person] Fixed incorrect message retrieval from the KB

alex-mitrevski committed:
[find_my_mates/interview_person] Added a missing rasa Interpreter import

ErickKramer committed:
Merge branch 'devel' of https://github.com/b-it-bots/mas_domestic_robotics into devel

ErickKramer committed:
Moved nlu model to mas_models

ErickKramer committed:
Modified the path to load nlu model from mas_models

ErickKramer committed:
Obtained the model_directory directly from the get_package_path function

alex-mitrevski committed:
[move_arm_action] Removed the roll_dmp script, which is now in the ros_dmp package

minhnh commented on mdr_planning/mdr_scenarios/mdr_robocup_tasks/mdr_where_is_this/ros/src/mdr_where_is_this/scenario_states/receive_information.py:
```diff
@@ -0,0 +1,128 @@
+        # Setup the listen goal
+        goal = ListenGoal()
+
+        if not self.number_of_retries == 0:
```
`self.number_of_retries` is never decremented here. I would remove the retry logic if it's meant to loop forever.

minhnh replied:
@PatrickNa can you clarify what you meant to do here? I believe you wrote this right?

alex-mitrevski committed:
Fixed various linter issues in multiple packages

alex-mitrevski committed:
[find_people] Added missing imports and fixed additional linting issues

alex-mitrevski commented on mdr_planning/mdr_actions/mdr_perception_actions/mdr_find_people/ros/src/mdr_find_people/action_states.py:
```diff
@@ -0,0 +1,111 @@
+from find_people import FindPeople
+
+
+POINTCLOUD_TOPIC = '/hsrb/head_rgbd_sensor/depth_registered/rectified_points'
```
Needs to be passed as a parameter to the action

minhnh replied:
related open issues #152 #172

argenos replied:
This is in #193

alex-mitrevski commented on mdr_planning/mdr_actions/mdr_perception_actions/mdr_find_people/ros/src/mdr_find_people/action_states.py:
```diff
@@ -0,0 +1,111 @@
+
+
+    @staticmethod
+    def is_inside_arena(pose):
```
Should be removed since it's super-hardcoded

argenos replied:
Before we merge or should we open an issue for this? Probably should be refactored to use the topological map

minhnh replied:
#152 and #172 are open I think. You can add a comment there.

argenos replied:
Done! Should I mark this as resolved then?

minhnh replied:
I'm doing it

minhnh commented on mdr_planning/mdr_behaviours/mdr_hri_behaviours/common/models/checkpoint:
```diff
@@ -0,0 +1,2 @@
+model_checkpoint_path: "/home/patrick/rasa/models/current/nlu/intent_classifier_tensorflow_embedding.ckpt"
```
@PatrickNa these still need refactoring

PatrickNa replied:
What is the purpose of this file anyway?

argenos replied:
@ErickKramer can you help out with this?

ErickKramer replied:
This file is generated when executing the training script from Rasa. I don't think it has any role after training is done.

minhnh commented:
`mdr_planning/mdr_behaviours/mdr_hri_behaviours/common/models/training_data.json` still need to be removed/refactored I believe.

mas-hsr committed:
[where_is_this] remove extra spoken sentences

mas-hsr committed:
[where_is_this] change delays for natural spoken speed

alex-mitrevski committed:
[move_arm_action] Added ros_dmp to the list of dependencies

alex-mitrevski committed:
[move_arm_action] Added a functionality for preempting the action

alex-mitrevski committed:
[move_arm_action] Added an action preemption test

alex-mitrevski committed:
[move_arm_action/action_states] I was using an incorrect var name for preemption setting

alex-mitrevski committed:
[move_arm_action/server] The result is now set even in the case of preemption

alex-mitrevski committed:
[move_arm_action/dmp] Replaced the DMP service call with an API call

alex-mitrevski committed:
[move_arm_action/dmp] All DMP parameters are now configurable

alex-mitrevski committed:
[move_arm_action/dmp] The velocity limit is now also parameterised

argenos commented:
@alex-mitrevski @PatrickNa what is missing to merge this? Could you make your reviews please?

alex-mitrevski commented:
There are at least a couple of open comments that should be resolved before we merge (e.g. an HSR-specific topic being hard-coded or incorrect model paths being specified).

PatrickNa commented on mdr_environments/go-2019/map.yaml:
```diff
@@ -0,0 +1,7 @@
+image: map.pgm
```
All the go-2019 stuff we are taking as example and with the eventuality that parts of it could be useful in the next German Open, right?  Same holds true for the Rheinbach configuration files?

Maybe we could keep them in a separate repo like the models?

argenos replied:
I’m not sure why the maps cause this reaction, but sure I can move them somewhere else. 

I don’t think that a single repo for such small files makes sense.  In a previous review I mentioned they can be used for testing and, in the case of go-2019, we can probably have a base map to start with and might not need to remap next year. 

I would reverse the question, why would we want to delete other maps?

PatrickNa replied:
I'm not saying delete. But those models and configurations that might or might not be valid to us, shall we not separate them from the functional code? We have the models repo, which could also include stuff like that.

However I am not that determined about that. I'm also fine leaving it as it is for now.

argenos replied:
Based on our talk today: 
- Add a README to describe the different maps
- Move the maps once we create the config repo for the scenarios as well

argenos replied:
Solved the README in cef25da

PatrickNa commented on mdr_planning/mdr_behaviours/mdr_hri_behaviours/common/models/metadata.json:
```diff
@@ -0,0 +1,97 @@
+{
```
Should not all this model data go into mas_models?

argenos replied:
Yep,  I thought there was a PR for this already.

argenos replied:
There is no PR, but there is an issue (#171) which is assigned to you, could you move them there when you get the chance?

argenos committed:
mdr_environments: Rename go-2019 map to german_open_arena

argenos committed:
mdr_environments: Add README

argenos committed:
mdr_move_arm_action: Add ros_dmp to rosinstall

argenos committed:
mdr_find_people: Remove HSR-specific topic

argenos commented:
To clarify the missing to-dos, please correct me if there's something missing :

- [x] Removing the models form `mdr_hri_behaviours` #171 - assigned to @PatrickNa 

- [x] Removing HSR-specific topics - PR #193 

- [x] Fixing the build in `devel` - @alex-mitrevski, I added the missing `ros_dmp` package to the rosinstall, but the build is still failing since you have missing dependencies.

argenos committed:
travis: Add wait time for docker push

argenos committed:
Merge branch 'devel' into fix/mdr_find_people-cleanup

argenos committed:
Merge pull request #193 from argenos/fix/mdr_find_people-cleanup

mdr_find_people cleanup

ErickKramer committed:
Merge branch 'devel' of https://github.com/b-it-bots/mas_domestic_robotics into devel

ErickKramer committed:
mdr_hri_behaviours nlu model is loaded from mas_models

ErickKramer committed:
nlu model moved to mas_models

Commit:
Clean and improve the speech matching functionality.

Instead of checking matches in two hard coded text files. The algorithm now loops over all files specified in the config directory. Additionally dependencies in the package.xml have been clarified and unnecessary comments removed. TODO: change the type of the msg of this package.

argenos committed:
mdr_speech_matching: Comment out dependency to fuzzywuzzy

argenos committed:
Merge branch 'devel' into refactoring/moving_mdr_hri_behaviors_models

argenos committed:
Merge pull request #194 from ErickKramer/refactoring/moving_mdr_hri_behaviors_models

Moving mdr_hri_behaviours nlu model to mas_models

argenos commented:
@alex-mitrevski @PatrickNa checks are passing and final changes were just merged, could you please review/approve so I can merge?

argenos committed:
mdr_hri_behaviors: Remove unused imports
